osquery/core/watcher.cpp

```diff
@@ -128,8 +138,25 @@ CLI_FLAG(uint64,
 
 CLI_FLAG(bool, disable_watchdog, false, "Disable userland watchdog process");
 
+HIDDEN_FLAG(bool,
+            enable_watchdog_logging,
```

Comment:
If I understand the intent right, this logs every few seconds. Which feels like it's debug/developer oriented, and not something one might want to enable in production. As such, I'd suggest calling this flag `watchdog_verbose` or `watchdog_debug`. (My bias is to `debug`, but we appear to have prior art for both.)